src/python/turicreate/test/test_object_detector.py

```diff
@@ -150,7 +150,7 @@ def setUpClass(self):
             self.get_ans['grid_width'] = lambda x: x > 0
             self.get_ans['training_mean_average_precision'] = lambda x: True
             self.get_ans['random_seed'] = lambda x: True
-            self.get_ans['training_average_precision'] = lambda x: x >= 0
+            self.get_ans['training_average_precision'] = lambda x: isinstance(x, dict)
```

Comment:
give any explanation? the 2 lambdas are functionally different.

Reply:
Yes, it seems like the correct type of x should be a dictionary instead of a single value.

Reply:
'training_average_precision' contains ap for each class (dict).
'training_mean_average_precision' is the mean of each ap which should be a single value.

Reply:
Could we also check that the keys are the class labels, and that the values are numeric?

Reply:
so you mean the data representation is wrong. but how could a `dict` compare with int? How could it pass the test pipeline? What happens in our test pipeline? Do we need to hire `the real ghostbusters` to clear the ghost in our test machines? Should I say happy Halloween?
```
>>> dict() > 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'dict' and 'int'
```

Reply:
sound like a good idea will add it